test.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+    @brief The program implements a check of the function operation quadro_solve.cpp
+    @version 1.0.0
+*/
+
+#include "quadro_solve.cpp"
+
+/**
+ * @brief A function for reading float values from terminal
+ * @version 1.0.0
+ * @param a
+ * @param b
+ * @param c
+ * @return Nothing
+ */
+
+void readCoefficients(double* a, double* b, double* c) {
+    assert(a != NULL);
+    assert(b != NULL);
+    assert(c != NULL);
+
+    printf("Введите ТРИ числа:\n");
+    while (scanf("%lg %lg %lg", a, b, c) != 3) {
+        printf("Вы не ввели число\n");
```

Comment:
I could enter "5 10 c" and get this message. Probably you should change it a bit